include/z64actor.h

```diff
@@ -329,6 +329,8 @@ typedef struct
     /* 0x15A */ u16 unk_15A;
     /* 0x15C */ u32 unk_15C;
     /* 0x160 */ u8 unk_160;
+	/* 0x161 */ char unk_161;
```

Comment:
It looks like your editor is inserting tabs instead of spaces? Same issue in the actor's header